site/docs/.vuepress/components/webhook-util/translations/en_US.ts

```diff
@@ -0,0 +1,72 @@
+import { Translation } from "./types";
+
+export const lang: Translation = {
+  tokenCard: {
+    disclaimer: {
+      title: "About your token",
+      content:
+        "We never store your bot's token in the backend. It is stored in your browser's localStorage. Do not go around giving your token to random websites, no matter how nicely they ask!",
+    },
+    fields: {
+      token: {
+        label: "Bot Token",
+        placeholder: "Obtained from talking to @botfather",
+      },
+    },
+    buttons: {
+      loadBotInfo: "Load bot info",
+    },
+  },
+  botCard: {
+    canJoinGroups: "Can join groups",
+    canReadGroupMessages: "Can read all group messages",
+    inlineQueries: "Supports inline queries",
+    tabs: {
+      webhookInfo: "Webhook Info",
+      manageWebhook: "Manage Webhook",
+    },
+    buttons: {
+      changeToken: "Change token",
```

Comment:
```suggestion
      changeToken: "Change Token",
```

Reply:
jsyk, button text is full caps always

Reply:
I already know. It must be consistent in the source code as well :)